lib/audited/audit.rb

```diff
@@ -178,8 +178,7 @@ def self.collection_cache_key(collection = all, *)
     private
 
     def set_version_number
-      max = self.class.namespaced.not_before_created_at(auditable).auditable_finder(auditable_id, auditable_type).maximum(:version) || 0
-      self.version = max + 1
+      self.version = DateTime.now.strftime('%Q').to_i
```

Comment:
will that make an update SQL query?

Reply:
Not in this line of code, but overall it will be only inserts into database